sqldb/sqlerrors_js.go

```diff
@@ -0,0 +1,86 @@
+//go:build js || (windows && (arm || 386)) || (linux && (ppc64 || mips || mipsle || mips64))
+
+package sqldb
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/jackc/pgconn"
+	"github.com/jackc/pgerrcode"
+)
+
+var (
+	// ErrRetriesExceeded is returned when a transaction is retried more
+	// than the max allowed valued without a success.
+	ErrRetriesExceeded = errors.New("db tx retries exceeded")
+)
+
+// MapSQLError attempts to interpret a given error as a database agnostic SQL
+// error.
+func MapSQLError(err error) error {
```

Comment:
👍 

Missing the `sqlite` mapping?

Reply:
Yes this is for wasm, so we can't really depend on sqlite here.